web/README.md

```diff
@@ -0,0 +1,9 @@
+# Fashion Tips Web Client
+The Web client to Fashion Tips Bionic University project.
+
+# Running locally
+`git clone https://github.com/FashionTips/fashion_tips.git`
+`cd web`
+`mvn jetty:run`
```

Comment:
This is a point - we do not. The only things the user has to have at his local machine are git and maven. That's all. He will use maven embedded jetty server to run the app.

We can use embedded tomcat, instead of jetty, as well.